src/SecretManagement.KeyChain/SecretManagement.KeyChain.psd1

```diff
@@ -1,13 +1,13 @@
 @{
-    ModuleVersion = '0.1.1'
+    ModuleVersion = '0.1.2'
     CompatiblePSEditions = @('Core')
     GUID = '74bb5212-2a5d-451d-8f43-edf9bcd2efe8'
     Author = 'Steve Lee'
     CompanyName = 'Microsoft Corporation'
     Copyright = '(c) Microsoft Corporation'
     Description = 'SecretManagement extension vault for macOS KeyChain'
     RootModule = 'SecretManagement.KeyChain.psm1'
-    # RequiredModules = @('Microsoft.PowerShell.SecretManagement')
+    RequiredModules = @('Microsoft.PowerShell.SecretManagement')
```

Comment:
This is deliberately commented out for now as PowerShellGet has a problem with resolving dependencies on preview modules.  Need to keep this commented out until SecretManagement is no longer a prerelease.